### PR TITLE
Slawomir/auth responsive

### PR DESCRIPTION
**Pull request by thewilkupl**
This pull request contains:

- created Assets folder
- moved GlobalStyles.js to Assets/style
- created breakpoints in device.js
- applied responsive style to Auth and Form Components

### CODE REVIEW

File: frontend/src/assets/styles/device.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+};
+
+export const device = {
+  mobileS: `(max-width: ${size.mobileS})`,
+  mobileM: `(max-width: ${size.mobileM})`,
+  mobileL: `(max-width: ${size.mobileL})`,
+  tablet: `(max-width: ${size.tablet})`,
+  laptop: `(max-width: ${size.laptop})`,
+  laptopL: `(max-width: ${size.laptopL})`,
+};
```
Comment (Willaiem) [inline frontend/src/assets/styles/device.js:17]: Change the "size" and "device" to "sizes" and "devices" (typo).

Comment (Willaiem) [reply frontend/src/assets/styles/device.js:17]: This file could be just a TypeScript file - no need to make it as .js.

**Review by Willaiem — CHANGES_REQUESTED** [review]
I added the suggestions before merging.

Also, on Samsung S5 (and any devices with less than 360px width) there is a bug related to responsiveness (the logo is partly or almost out of the screen): https://ibb.co/CtFPNKJ
The same happens for 540px width.

I "fixed" that by removing the "height" from AuthWrapper, but I am unable to suggest, which part of CSS "breaks" on these resolutions.

**Comment by thewilkupl** [comment]
I think I have removed the 'bug' related to height
https://ibb.co/CJ8Fjks
https://ibb.co/6yvBk25

**Pull request merged**